docs/service-manifest.md

```diff
@@ -95,6 +95,18 @@ spec:
 
 مقدار پیشفرض برای پروتکل TCP است، پس نیازی نیست همیشه قید کنید اگر به UDP نیازی ندارید.
 
+ - **volume_mounts**
+volume_mounts یک لیست از آدرس های محل ذخیره سازی می باشد که به volume اصلی mount می شوند و هر container می تواند دیتاهای خود را در محلی که در manifest قرار داده شده است نوشته و ذخیره کند.
+مثلا فرض کنید سرویس شما می خواهد داده های مربوط به پایگاه داده خود را که در اینجا MySQL فرض شده است را از این به بعد در volume اصلی فندق یا همان pv-storage ٬ بر روی آدرس جدیدی به نام var/lib/mysql/ ذخیره سازی کند برای همین sub_path مربوط به mysql از pv-storage فندق را به آدرس جدید mount میکنیم.
```

Comment:
Where you mentioned `pv-storage`, I think it's not right, user's volume's name is `{{namespace}}-volume`.
for now I think we should refer to volume just as **User's default volume**, since at this point users are not able to see the concept of volume from outside and they only have one volume